tests/performance_tests/performance_test_basic.py

```diff
@@ -77,18 +79,22 @@ def __init__(self, testHelperConfig: TestHelperConfig=TestHelperConfig(), cluste
         self.ptbLogDir = f"{self.rootLogDir}/{os.path.splitext(os.path.basename(__file__))[0]}"
         self.testTimeStampDirPath = f"{self.ptbLogDir}/{self.testStart.strftime('%Y-%m-%d_%H-%M-%S')}"
         self.trxGenLogDirPath = f"{self.testTimeStampDirPath}/trxGenLogs"
+        self.varLogsDirPath = f"{self.testTimeStampDirPath}/var"
+        self.etcLogsDirPath = f"{self.testTimeStampDirPath}/etc"
+        self.etcEosioLogsDirPath = f"{self.etcLogsDirPath}/eosio"
         self.blockDataLogDirPath = f"{self.testTimeStampDirPath}/blockDataLogs"
         self.blockDataPath = f"{self.blockDataLogDirPath}/blockData.txt"
         self.blockTrxDataPath = f"{self.blockDataLogDirPath}/blockTrxData.txt"
         self.reportPath = f"{self.testTimeStampDirPath}/data.json"
-        self.nodeosLogPath = "var/lib/node_01/stderr.txt"
 
         # Setup Expectations for Producer and Validation Node IDs
         # Producer Nodes are index [0, pnodes) and validation nodes/non-producer nodes [pnodes, _totalNodes)
         # Use first producer node and first non-producer node
         self.producerNodeId = 0
         self.validationNodeId = self.clusterConfig.pnodes
 
+        self.nodeosLogPath = f"var/lib/node_0{self.validationNodeId}/stderr.txt" if self.validationNodeId < 10 else f"var/lib/node_{self.validationNodeId}/stderr.txt"
```

Comment:
No need for the conditional.  `self.nodeosLogPath = f'var/lib/node_{self.validationNodeId.zfill(2)}/stderr.txt'`

Reply:
Addressed: [abc5c41](https://github.com/AntelopeIO/leap/pull/393/commits/abc5c4150d3fc732a8d6c51bfe555911d89d270b)